src/adagrams.js

```diff
@@ -1,7 +1,115 @@
 const Adagrams = {
+  shuffle(array) {
+    let currentIndex = array.length;
+    let temporaryValue, randomIndex;
+
+    // While there remain elements to shuffle...
+    while (0 !== currentIndex) {
+    // Pick a remaining element...
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex -= 1;
+
+      // And swap it with the current element.
+      temporaryValue = array[currentIndex];
+      array[currentIndex] = array[randomIndex];
+      array[randomIndex] = temporaryValue;
```

Comment:
You seem to have pulled this from Stack Overflow! When taking a large chunk of code, it's good to credit and link it!